Fix CI

rhaschke opened the pull request:
Travis has several issues running the unit tests. I migrated to GitHub actions finally. 
#6 fixes the [remaining issues](https://github.com/rhaschke/PyLMI-SDP/actions/runs/1647150141) in GHA.

cdsousa committed:
Fix new SymPy expected printings.

rhaschke committed:
Drop unsupported option --use-mirrors from pip install

rhaschke committed:
Try other python + sympy combinations

- Update to python 3.6: 3.3 is not available anymore
- Add sympy 0.7.6

rhaschke committed:
Migrate to GitHub Actions

rhaschke committed:
Adapt expected SDPA output

zero is formatted as 0.0 instead of 0.

cdsousa commented:
Hi @rhaschke ! It is great to see that someone is fixing long pending issues in the package, as I myself am not maintaining it.
It seems to me that it is worth letting you fix/evolve the package as you see fit, so I'm giving you access to it (I think you can do everything, if not, we can think of moving the repo somewhere else).

So please, be my guest and hit the "merge PR" buttons on the PRs!

rhaschke commented:
Thanks for your trust. I will be happy to merge my fixes and also cut a new release if desired.
However, I won't be able to maintain the package in a long run.

rhaschke committed:
pytest: include doctest + coverage

rhaschke committed:
Drop Travis

rhaschke committed:
numpy.matrix -> numpy.array

to remove PendingDeprecationWarning:
Please adjust your code to use regular ndarray.

rhaschke committed:
CI: split pytest + dependency installation

rhaschke committed:
Disable doctest for specific sympy versions

Some sympy versions yield a different formatting of float numbers,
e.g. 0 vs 0.0, which makes the doctest fail.

cdsousa commented:
> Thanks for your trust. I will be happy to merge my fixes and also cut a new release if desired.

Feel free to do what you need.

> However, I won't be able to maintain the package in a long run.

Anyway, these PRs are absolutely better than nothing at all 😊 Thank you very much!
